grub-mkrelpath: add page (#9301)
tldr-pages/tldr: pages/linux/grub-mkrelpath.md

# grub-mkrelpath

> > Make a file system path relative to the root of its file system.

Reviewer: ```suggestion
> Make a filesystem path relative to the root of its filesystem.
```

> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.

> - Set the filesystem path relative to the root which can be used by other GRUB utilities:

Reviewer 2: ```suggestion
- Set a specific filesystem path relative to the root which can be used by other GRUB utilities:
```

Optional fix: `a specific` may sound better.

> - Set the file system path relative to the root which can be used by other GRUB utilities:

Reviewer: ```suggestion
- Set the filesystem path relative to the root which can be used by other GRUB utilities:
```

> `grub-mkrelpath {{path/to/relativepath}}`

Reviewer: this should be a directory, right?
```suggestion
`grub-mkrelpath {{path/to/directory}}`
```

Reviewer 3: I think we should leave the relative path as it is. 

Ref:

```text
The program grub-mkrelpath makes a file system path relative to the root of its containing file system. For instance, if /usr is a mount point, then:

$ grub-mkrelpath /usr/share/grub/unicode.pf2
‘/share/grub/unicode.pf2’
```

Reviewer: It's ambigous, it's a `path/to/file_or_directory` then?

Author: > It's ambigous, it's a `path/to/file_or_directory` then?

It makes sense and clear for others. Shall we go with this change @kbdharun and @navarroaxel ?

Reviewer 3: > It's ambigous, it's a `path/to/file_or_directory` then?

I am not sure should we ask @CleanMachine1 ?

Author: > It's ambigous, it's a `path/to/file_or_directory` then?

@navarroaxel, I have found the following link where the path is mentioned as file name. https://manpages.ubuntu.com/manpages/jammy/man1/grub-mkrelpath.1.html

Should we make it as path_to_file?

> - Print the GRUB version number:

Reviewer 3: ```suggestion
- Display GRUB version number:
```

`grub-mkrelpath --version`

> - Print help page:

Reviewer 3: ```suggestion
- Display help:
```

> - Display GRUB version number:
> 
> `grub-mkrelpath --version`
> 
> - Display help:
> 
> `grub-mkrelpath --help`

Reviewer 2: ```suggestion
- Display help:

`grub-mkrelpath --help`

- Display GRUB version number:

`grub-mkrelpath --version`
```

Optional fix: examples can be reordered.
Hint: to find all pages containing `--help` and `--version` you can use `for f in *.md; do if grep -E -- --version "$f" &> /dev/null && grep -E -- help "$f" &> /dev/null; then echo $f; fi; done` one liner.
